how-to/customize-workspace/scripts/package-config.json

```diff
@@ -0,0 +1,29 @@
+{
+	"manifestHost": "http://localhost:8080",
+	"contentPacks": [
+		{
+			"id": "public",
+			"sourceRoot": "./public",
+			"sources": ["manifest.fin.json", "manifest-hosts.json"],
```

Comment:
this doesn't seem to work if you pick second.manifest.fin.json

Reply:
Fixed. Also the manifest is now a command line parameter, so you can kickoff the packaging without needing to modify the config file (defaults to manifest.fin.json).

```shell
npm run package-content --manifest=second.manifest.fin.json --env=uat --host=https://openfin.mydomain.com
```